cloud-aws/src/main/resources/templates/aws-cf-stack-freeipa.ftl

```diff
@@ -120,7 +120,10 @@
         <#else>
         "VPCZoneIdentifier" : [{ "Ref" : "SubnetId" }],
         </#if>
-        "LaunchConfigurationName" : { "Ref" : "AmbariNodeLaunchConfig${group.groupName?replace('_', '')}" },
+        "LaunchTemplate" : {
+          "LaunchTemplateId": { "Ref" : "AmbariNodeLaunchTemplate${group.groupName?replace('_', '')}" },
```

Comment:
AmbariNodeLaunchTemplate -> CDPNodeLaunchTemplate

Reply:
or rather ClusterManagerNodeLaunchTemplate

Reply:
Both are fine but cf template has a limit on the length this is why i think shorter is better

Reply:
Renamed it to ClusterManagerNodeLaunchTemplate, ran just fine